app/views/demos/show.html.erb

```diff
@@ -1,5 +1,11 @@
 <% content_for :demo do %>
-  <script src="<%= @script_path %>" async="async"></script>
+  <script>
+    document.addEventListener('turbolinks:load', function () {
+      var script = document.createElement('script');
+      script.src = "<%= @script_path %>";
+      document.head.appendChild(script);
+    });
+  </script>
```

Comment:
Would this be better handled by a Stimulus controller?

Reply:
I think it's a bit overkill